src/current/_includes/releases/v24.2/v24.2.10.md

```diff
@@ -0,0 +1,55 @@
+## v24.2.10
+
+Release Date: February 6, 2025
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-2-10-general-changes">General changes</h3>
+
+- The protected timestamp (PTS) records of running changefeeds are now updated when the set of targets changes, such as when system tables are added to the protected tables list. [cockroachdb/cockroach#138653][#138653]
+
+<h3 id="v24-2-10-sql-language-changes">SQL language changes</h3>
+
+- The `legacy_varchar_typing` session setting has been added, which reverts the changes of #133037 that cause the change in typing behavior described in #137837. Specifically, it makes type-checking and overload resolution ignore the newly added "unpreferred" overloads. This setting defaults to `on`. [#138002][#138002]
+- Since v23.2 table statistics histograms have been collected for non-indexed JSON columns. Histograms are no longer collected for these columns if the `sql.stats.non_indexed_json_histograms.enabled` cluster setting is set to `false`. This reduces memory usage during table statistics collection, for both automatic and manual collection via `ANALYZE` and `CREATE STATISTICS`. [#140267][#140267]
```

Comment:
```suggestion
- Since v23.2 table statistics histograms had been collected for non-indexed JSON columns. Histograms are no longer collected for these columns if the `sql.stats.non_indexed_json_histograms.enabled` cluster setting is set to `false`. This reduces memory usage during table statistics collection, for both automatic and manual collection, via `ANALYZE` and `CREATE STATISTICS`. [#140267][#140267]
```

Reply:
added the edit locally (b/c i need to fix the inevitable releases.yml merge conflict!), will push momentarily